### PR TITLE
SwiftGen implementation

### PR DESCRIPTION
**Pull request by AlexYanski**
-[x] Assets generation
-[x] Strings generation

### CODE REVIEW

File: Bootstrap/Generated/Strings.swift [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+// swiftlint:disable superfluous_disable_command
+// swiftlint:disable file_length
+
+// MARK: - Strings
+
+// swiftlint:disable explicit_type_interface function_parameter_count identifier_name line_length
+// swiftlint:disable nesting type_body_length type_name
+internal enum L10n {
+
+  internal enum NavigationBar {
```
Comment (pkondrashkov) [inline Bootstrap/Generated/Strings.swift]: Can we prevent if being uppercased?

File: Podfile [hunk 2]
```diff
@@ -1,11 +1,12 @@
 # Uncomment the next line to define a global platform for your project
-platform :ios, '10.0'
 
+platform :ios, '10.0'
```
Comment (pkondrashkov) [inline Podfile]: Use 9.0 please

File: Bootstrap/Generated/Strings.swift (hunk 1 above)
Comment (AlexYanski) [reply Bootstrap/Generated/Strings.swift]: Could't find any information about that

File: Podfile (hunk 2 above)
Comment (AlexYanski) [reply Podfile]: Done

**Pull request merged**